src/Microsoft.Azure.ServiceBus/QueueClient.cs

```diff
@@ -78,7 +78,6 @@ public QueueClient(ServiceBusConnectionStringBuilder connectionStringBuilder, Re
         /// Instantiates a new <see cref="QueueClient"/> to perform operations on a queue.
         /// </summary>
         /// <param name="connectionString">Namespace connection string. Must not contain queue information.</param>
-        /// <param name="entityPath">Name of the queue</param>
```

Comment:
>/// <param name="entityPath">Name of the queue</param> [](start = 8, length = 54)

I think this should be left as is. Revert this. Not everyone might understand the 'path' here

Reply:
@nemakam, isn't that wrong? if the parameter is "entity path", it can't be an "entity name".

`org/department/queue` for example, the **path** is `org/department/queue`, but **name** is `queue`

If it's reverted, then the change should be
```
/// <param name="entityPath">Path of the queue</param>
```

Reply:
`org/department/queue`.. What scenario is this? what is `org` and `department` in this case?
For us, `queue` doesn't really have a path. It just has a name. Even `topic` has a name. But entities like `subscription` have paths.
So I would call `org/department/queue` as the queue *name*.

Reply:
This brings back [this issue](https://github.com/Azure/azure-service-bus-dotnet/issues/67#issuecomment-298722100) and what was describes back in May.

I think it will be confusing to see

https://github.com/Azure/azure-service-bus-dotnet/blob/364b48319987421b3e931cd63b048966391bd705/src/Microsoft.Azure.ServiceBus/QueueClient.cs#L121

and 

https://github.com/Azure/azure-service-bus-https://github.com/Azure/azure-service-bus-dotnet/blob/364b48319987421b3e931cd63b048966391bd705/src/Microsoft.Azure.ServiceBus/QueueClient.cs#L140

at the same time.

Reply:
If you look at the implementation of `QueueClient.Path`, we just do `return this.QueueName`. Now it brings us to, why have both? Coz we have a common parameter `Path` in every client entity (`MessageReceiver` to `QueueClient` to `SubscriptionClient` etc). But now, we have a `QueueClient` as a separate class which is aimed at people who potentially might not have a complete understanding of what `Path` means? Since this is a client specific to queues, we have a `QueueName` as an additional property. This property is especially required when you start thinking of say `SubscriptionClient` which needs to have two such properties `SubscriptionName` and `TopicName` along with the usual `Path` which are all different. Its just about having consistency across the board.